packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -244,6 +340,14 @@ moduleFor('Dynamic content tests (content position)', class extends DynamicConte
 
 });
 
+moduleFor('@htmlbars Dynamic content tests ("trusted" curlies)', class extends DynamicContentTest {
```

Comment:
These tests are currently [failing](https://travis-ci.org/emberjs/ember.js/jobs/120383033) on IE9/10/11. Tagged them for HTMLBars for now.